src/mono/msbuild/android/build/AndroidApp.targets

```
@@ -0,0 +1,174 @@
+<Project>
+  <UsingTask TaskName="AndroidAppBuilderTask" 
+             AssemblyFile="$(AndroidAppBuilderTasksAssemblyPath)" />
+  <UsingTask TaskName="NetTraceToMibcConverter" 
+             AssemblyFile="$(NetTraceToMibcConverterTasksAssemblyPath)" />
+
+  <Target Name="AndroidBuildApp" AfterTargets="$(AndroidBuildAppAfterThisTarget)" />
+
+  <Target Name="_AndroidCoreBuild" BeforeTargets="AndroidBuildApp" DependsOnTargets="$(AndroidBuildAppDependsOn)" />
```

Do we need a separate _AndroidCoreBuild target?
```suggestion
  <Target Name="AndroidBuildApp" DependsOnTargets="$(AndroidBuildAppDependsOn)" />
```
And if we really want to keep `Publish` to occur after `AndroidBuildApp`, can add `AfterTargets="$(AndroidBuildAppAfterThisTarget)"`.